src/governance.h

```diff
@@ -238,7 +238,10 @@ class CGovernanceManager
     // keep track of the scanning errors
     object_m_t mapObjects;
 
-    count_m_t mapSeenGovernanceObjects;
+    // mapDeletedGovernanceObjects contains key-value pairs, where
+    //   key   - governance object's hash
+    //   value - either object's status or expiration time for deleted objects
+    count_m_t mapDeletedGovernanceObjects;
```

Comment:
Fixed. But I left `int` type and renamed it to `hash_time32_m_t` (to be compatible with previous versions)